handlers/whatsapp/whatsapp.go

```diff
@@ -776,8 +776,12 @@ func (h *handler) getTemplate(msg courier.Msg) (*MsgTemplating, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "invalid templating definition")
 	}
+	// check country
+	if templating.Country != "" {
```

Comment:
This is set on all templating now in RapidPro isn't it? So for a US channel, it will be set to "US"

Reply:
No, the country isn't defined by channel, it's defined by the language in WhatsApp sync: https://github.com/nyaruka/rapidpro/pull/2836/files#diff-4714c50434646dabb7e2ae03726b788dR202

Just a quick review of behavior:
- On WhatsApp sync, the language is mapped from WhatsApp format to the fields `language` (iso639-3) and `country` (iso3166-2);
- On Courier, the language is mapped back to WhatsApp format;
- Examples:
```
('eng', None) -> 'eng'    -> 'en'
('eng', 'US') -> 'eng_US' -> 'en_US'
...
```
The entry `por` on language map, for example, was not removed, it was kept pointing to `pt_PT`.

Reply:
Ah, understand now, forgot country was coming from the template itself not from the channel. Ok!